requirements.txt

```diff
@@ -0,0 +1,4 @@
+asgiref==3.6.0
+Django==4.1.7
+sqlparse==0.4.3
+cosglupieg==21.12.1
```

Comment:
A co to za dziwny import?

Reply:
This is now corrected! Thanks for pointing out!